### PR TITLE
Create SimonVerhoeven.json

### PR DESCRIPTION
**Pull request by SimonVerhoeven**
Initial profile commit

## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/5943"></a>

### CODE REVIEW

**Review by sidharthviz — DISMISSED** [review]
LGTM 👍🏼

File: data/SimonVerhoeven.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    },
+    {
+      "name": "Follow me on Mastodon",
+      "url": "https://twitter.com/simon_verhoeven",
+      "icon": "FaMastodon"
+    },
+    {
+      "name": "LinkedIn: let's connect",
+      "url": "https://www.linkedin.com/in/simonverhoeven/",
+      "icon": "FaTwitter"
```
Comment (mahfuztbt) [inline data/SimonVerhoeven.json:27]: here you put your linkenIn link but you add twitter icon. please correct it to `FaLinkedin` in line 27

**Review by mahfuztbt — CHANGES_REQUESTED** [review]

File: data/SimonVerhoeven.json (hunk 1 above)
Comment (SimonVerhoeven) [reply data/SimonVerhoeven.json]: Thank you

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Welcome to LinkFree 🎉

**Pull request merged**